terraso_backend/apps/graphql/schema/sites.py

```diff
@@ -62,10 +66,15 @@ class Input:
 
     @classmethod
     def mutate_and_get_payload(cls, root, info, **kwargs):
+        log = audit_log_services.new_audit_logger()
```

Comment:
suggestion: Could you add this as a class variable on the parent class? Will let us share that code between different methods.